Consul.Test/ACLReplicationTest.cs

```diff
@@ -23,24 +23,8 @@
 
 namespace Consul.Test
 {
-    public class ACLReplicationTest : IDisposable
```

Comment:
Can we create separate PR for `*.cs` changes, it will make the matrix PR more coherent.

Reply:
Sure

Reply:
https://github.com/G-Research/consuldotnet/pull/88 done